src/styles/_typography.scss

```diff
@@ -48,8 +48,7 @@ h6 {
 }
 
 // Text and long form content
-ul li,
-p {
+ul li {
   font: var(--gcds-font-text);
   max-width: 65ch;
 }
```

Comment:
Instead of removing the paragraph here, can we please add a media query to load in the mobile font once the viewport is smaller than 48em? You can [copy the code](https://github.com/cds-snc/gcds-components/blob/main/packages/web/src/components/gcds-text/gcds-text.css#L24) from the text component.

Reply:
The CSS for this is already in the utility framework's CSS (that's what the removed style was overriding), do we want to duplicate that with this custom CSS?

Reply:
Oh gotcha, now I'm with you - sorry! No, you are correct - let's remove it! If you have time and feel like it: can you check if we have other duplicate styles here that we already have in the UF? Or otherwise, add a cleanup ticket? Could be a good cleanup task for the Christmas break. :)